intro-to-tests/problem-set-intro-to-pytest.md

```diff
@@ -92,7 +93,21 @@ In tests, long test names are welcome. For each test case, the best test names u
 
 The following guideline is challenging to master, but the best test names tend to describe a situation, and not name exact variable names. If the variable name changes, then the test name could become inaccurate.
 
-Pytest will only recognize test names begin with `test_` or end with `_test`.
+Pytest will only recognize test names beginning  with `test_` or ending with `_test`.
+
+##### !end-hint
+
+##### !explanation
+
+For testing whether our function `calculate` returns a `TypeError` when we get an input that is not a number, the test name `test_calculate_non_numbers_returns_type_error()` is the most preferred name because it is specific and adds context describing which condition the test is checking. 
+
```

Comment:
Add explicit line break tag since Learn squashes blanks in explanations. In some places, additional blank lines around the the line break is needed to get the desired break between paragraphs, so I generally do it anywhere I use a line break.
```suggestion

<br />

```